frontend/src/pages/modelServing/screens/projects/utils.ts

```diff
@@ -176,3 +182,24 @@ export const useCreateInferenceServiceObject = (
 
   return createInferenceServiceState;
 };
+
+export const getProjectModelServingPlatform = (
+  project: ProjectKind,
+  disableKServe: boolean,
+  disableModelMesh: boolean,
+) => {
+  if (project.metadata.labels[KnownLabels.MODEL_SERVING_PROJECT] === undefined) {
+    if ((!disableKServe && !disableModelMesh) || (disableKServe && disableModelMesh)) {
+      return undefined;
+    }
```

Comment:
would it not make sense to include extra enum literals for both and none ?

Reply:
Yeah I think you are right, I could update it later.

Reply:
It's not actually easy to get `both` in, we also used this type in CustomServingRuntimes, which will need to return `[ServingRuntimePlatform.SINGLE, ServingRuntimePlatform.MULTI]`

Reply:
@DaoDaoNoCode ok i see how the array is used in the `Template` annotation.
Keep as is for now.